src/main/java/eggventory/Eggventory.java

```diff
@@ -44,10 +39,23 @@ public static void main(String[] args) {
         storage = new Storage(stockFilePath, stockTypesFilePath, loanListFilePath, personListFilePath,
                 templateListFilePath);
         parser = new Parser();
-        stockList = storage.load();
-        loanList = storage.loadLoanList();
-        personList = storage.loadPersonList();
-        templateList = storage.loadTemplateList();
+        stateInterface = new StateInterface(storage.load(), storage.loadLoanList(), storage.loadPersonList(),
+                storage.loadTemplateList());
+
+
+
+        /*
+        Calendar date = Calendar.getInstance();
+        loanList.addLoan("R500", "A123", 100, date, date);
+        loanList.addLoan("R500", "A6000", 100, date, date);
+        loanList.addLoan("ARDUINO", "A123", 100, date, date);
+        loanList.addLoan("NO", "A12", 100, date, date);
+
+        System.out.print("All: \n" + loanList.printLoans());
+        System.out.print("A123: \n" + loanList.printPersonLoans("A123"));
+        System.out.print("R500: \n" + loanList.printStockLoans("R500"));
+        */
+
```

Comment:
delete this again pls .. i deleted in a previous PR already

Reply:
aite